packages/scoutgame/src/protocol/resolveTokenOwnership.ts

```diff
@@ -69,18 +72,22 @@ export async function resolveTokenOwnership({
   week: ISOWeek;
   chainId: number;
 }): Promise<TokenOwnership> {
-  const lastBlock = await getLastBlockOfWeek({ week, chainId });
-  const standardNftContractAddress = getNFTContractAddress(week);
-  const starterNftContractAddress = getStarterNFTContractAddress(week);
+  const toBlock = await getLastBlockOfWeek({ week, chainId });
+  const season = getCurrentSeasonStart(week);
```

Comment:
there was actually a critical bug, we were getting nft contract by week, but it throws an error if it is not provided a season